sdk/nodejs/resource.ts

```diff
@@ -311,14 +312,14 @@ export const testingOptions = {
  * value as well as the Resource the value came from.  This allows for a precise 'Resource
  * dependency graph' to be created, which properly tracks the relationship between resources.
  */
-export class Output<T> {
+class OutputImpl<T> {
```

Comment:
this is not a break.  We still export an `Output` value that can be used in the same way this class was used.  However, the actual exported `Output` type is now quick a bit more complex than just a class (because it contained the proxied members).